less/inc/images.less

```diff
@@ -28,6 +28,9 @@
         -o-object-fit: scale-down;
         object-fit: scale-down;
       }
+      figcaption {
```

Comment:
This selector is `.wp-block-gallery.is-cropped .blocks-gallery-item figcaption` but not all galleries are `.is-cropped`.